watcher/registryhandler.go

```diff
@@ -114,19 +117,22 @@ func (ch *RegistryCommandsHandler) Start() {
 			} else if len(payload) > 0 {
 				status.Payload = payload
 			}
-			ch.patchCommandStatus(&cmd, status)
 
+			logger.L().Ctx(ctx).Info(fmt.Sprintf("finished handling %s command ", cmd.Spec.CommandType))
```

Comment:
actually we only use .Ctx(ctx) when we want to enrich the open telemetry span (and the ingestion is broken in our backend for the moment) can you remove them unless you log in Warning, Error or Fatal ?